packages/intl-getcanonicallocales/tsconfig.json

```diff
@@ -2,8 +2,6 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "lib": ["dom", "es5", "esnext.intl", "es2017.intl", "es2018.intl"],
-    "outDir": "dist",
-    "rootDir": "src"
-  },
-  "include": ["src"]
+    "outDir": "dist"
```

Comment:
Do we need `outDir`? Can we consolidate them in `ts_project` rule?

Reply:
yeah this is noop